test/bdd/pkg/vc-devapi/vc_devapi_steps.go

```diff
@@ -87,7 +93,8 @@ func (s *Steps) httpGet(url string, withAuth bool) error {
 	s.responseBody = body
 	s.responseStatus = resp.StatusCode
 
-	logger.Infof("body : %v and status %v", string(body), s.responseStatus)
+	logger.Info("httpGet content", log.WithAdditionalMessage(fmt.Sprintf("body: %s", string(body))),
```

Comment:
AdditionalMessage seems to be a bucket for leftovers. I think you should use WithResponseBody(body) here.